package.json

```diff
@@ -53,7 +53,7 @@
         "rimraf": "^3.0.2",
         "ts-node": "^9.0.0",
         "typescript": "^4.0.2",
-        "vsce": "^1.79.5",
+        "vsce": "^1.93.0",
```

Comment:
It should be ok to upgrade, we have never had any issues with vsce in the Metals extension.